mongodump/mongodump.go

```diff
@@ -430,6 +440,15 @@ func (dump *MongoDump) Dump() (err error) {
 	// we check to see if the oplog has rolled over (i.e. the most recent entry when
```

Comment:
This comment seems a bit far removed from its subject. Should be moved now that there’s the intervening barrier check?

Reply:
Done